docs/sphinx_qtile.py

```diff
@@ -38,6 +38,9 @@
 qtile_class_template = Template('''
 .. autoclass:: {{ module }}.{{ class_name }}
     :members: __init__
+    {% if is_widget %}
+    Supported bar orientations: {{ obj.orientations }}
```

Comment:
Does this just render the int (i.e. 0, 1, 2, 3?) or does it actually render HORIZONTAL or whatever? (Not that it's blocking merging this, but we should probably fix it at some point if it's just the int.)